src/main/service/abstracts/AbstractUser.java

```diff
@@ -0,0 +1,9 @@
+package main.service.abstracts;
+
+import main.entity.User;
+
+public abstract class AbstractUser {
+   public abstract void getRole(User user);
```

Comment:
add also here field for enum Role

Reply:
fixed

Reply:
Don't see the result :)
Add enum for role and add separate filed in AbstractUser for role